index.html

```diff
@@ -59,6 +59,38 @@
         </div>
         </div>
     </nav>
+    </nav>
+
+    <section class="hero flex column">
+        <div class="flex column">
+            <h1>Unsplash</h1>
+            <h2>The internet's source for visuals </h2>
+            <h2> Powered by creators everywhere. </h2>
+            <form class="relative text-input-with-icon hero_item--desktop-only">
+                <input class="text-input text-input-with-icon search-input" placeholder=" Search" type="text"
+                    name="search">
+                <svg class="text-input-with-icon" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512">
+                    <path
+                        d="M416 208c0 45.9-14.9 88.3-40 122.7L502.6 457.4c12.5 12.5 12.5 32.8 0 45.3s-32.8 12.5-45.3 0L330.7 376c-34.4 25.2-76.8 40-122.7 40C93.1 416 0 322.9 0 208S93.1 0 208 0S416 93.1 416 208zM208 352c79.5 0 144-64.5 144-144s-64.5-144-144-144S64 128.5 64 208s64.5 144 144 144z" />
+                </svg>
+                <input hidden type="submit" value="Search" />
+            </form>
+            <p class="hero_item--desktop-only"> Trending: <a href="https://unsplash.com/images/nature/flower"> flower,
+                </a> <a href="https://unsplash.com/wallpapers"> wallpapers, </a> <a
+                    href="https://unsplash.com/backgrounds">
+                    backgrounds, </a> <a href="https://unsplash.com/images/feelings/happy"> happy, </a> <a
+                    href="https://unsplash.com/images/feelings/love"> love </a> </p>
+        </div>
+
+        <p class="text-hero text-hero--bottom text-hero--left hero_item--desktop-only"><a
+                href="https://unsplash.com/photos/cEmftbsyyts"> Photo
+            </a> by <a href="https://unsplash.com/@michaelbaccin"> Michael Baccin </a>
+        <p class="text-hero text-hero--bottom text-hero--center hero_item--desktop-only"> Read more about <a
+                href="https://unsplash.com/license"> Unsplash
+                License
+            </a></p>
```

Comment:
We should have a `<footer>` in this section. Then we align the footer to be at the bottom of the here, instead of doing it for every individual item.